### ISSUE TITLE
Using `ParamSpec`, pyright shouldn't give error with a tuple but should give error with a list as a default value, following the docstring.

### ISSUE THREAD

**Issue by hyperkai** [microsoft/pyright#11192]
*Memo:
- pyright test.py
- pyright 1.1.407
- Python 3.14
- Windows 11

The docstring of [ParamSpec](https://docs.python.org/3/library/typing.html#typing.ParamSpec) shows two examples using the default value with a tuple as shown below:

```python
from typing import ParamSpec

print(help(ParamSpec))
```

> type IntFuncDefault[**P = (int, str)] = Callable[P, int]

> P = ParamSpec('P')
>      DefaultP = ParamSpec('DefaultP', default=(int, str))

But pyright gives the errors because a tuple is set instead of a list as shown below:

```python
                        # ↓ tuple  ↓
type IntFuncDefault[**P = (int, str)] = Callable[P, int] # Error
```

> error: Tuple expression not allowed in type expression
>    Use tuple[T1, ..., Tn] to indicate a tuple type or T1 | T2 to indicate a union type

```python
from typing import ParamSpec

P = ParamSpec('P')                     # ↓ tuple  ↓
DefaultP = ParamSpec('DefaultP', default=(int, str)) # Error
```

> error: Expected class but received "tuple[type[int], type[str]]" 

Actually, setting a list gets no error as shown below:

```python
from typing import ParamSpec

                        # ↓  list  ↓
type IntFuncDefault[**P = [int, str]] = Callable[P, int] # No error

P = ParamSpec('P')                     # ↓  list  ↓
DefaultP = ParamSpec('DefaultP', default=[int, str]) # No error
```

So using `ParamSpec`, pyright shouldn't give error with a tuple but should give error with a list as a default value, following the docstring.

**Comment by erictraut** [comment]
Pyright is working correctly here and is in compliance with the typing spec. ParamSpec defaults should be specified using square brackets (list syntax) rather than parens (tuple syntax). Refer to [PEP 696](https://peps.python.org/pep-0696/), which introduced TypeVar defaults to the type system.
